package.json

```diff
@@ -47,5 +48,8 @@
   "_moduleAliases": {
     "@": "dist"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.5.0",
+  "engines": {
+    "node": "18.13.0"
```

Comment:
It's really important to have this version of node, because older does not have `fetch` and `chatgpt` will throw an error